Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_controller.hpp

```diff
@@ -11,50 +11,44 @@
 #include "spdlog/spdlog.h"
 #include "../setup_helper.hpp"
 
+/// <summary>
+/// This class is designed as the object to control all object recognition networks and define corresponding functions 
+/// </summary>
 class MLController {
 
+	// The following two lines are for testing and demonstration, not used in the actual system
+
 	//std::vector<ModelInterface<rs2::video_frame>*> rgb_models;
 	//std::vector<ModelInterface<rs2::depth_frame>*> depth_models;
 
 	std::vector<ModelInterface*> models_;
 
-	/* concurrent? List of result objects?
-		Map<frame_number?, list of result objects>
-	*/
 	tbb::concurrent_vector<ClassificationResult> results_;
 
 public:
-
-	// Get's the results and clears them, so that they are not used again
+	/// <summary>
+	/// Get's the results and clears them, so that they are not used again
```

Comment:
What results? Why shouldn't they be used again?
Maybe add a more clear and descriptive description 😄